### PR TITLE
sys-apps/bleachbit-3.2.0_p20200308: upgraded to python3

### PR DESCRIPTION
**Pull request by lg188**
Bug: https://bugs.gentoo.org/711962
Signed-off-by: Lander Ghekiere <lander.ghekiere@gmail.com>

### CODE REVIEW

File: sys-apps/bleachbit/bleachbit-3.2.0_p20200308.ebuild [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
```
Comment (tietzc) [inline sys-apps/bleachbit/bleachbit-3.2.0_p20200308.ebuild]: Can you test if it works with 3.7 and 3.8 as well?

File: sys-apps/bleachbit/bleachbit-3.2.0_p20200308.ebuild [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+PYTHON_REQ_USE="sqlite(+)"
+DISTUTILS_SINGLE_IMPL=1
```
Comment (tietzc) [inline sys-apps/bleachbit/bleachbit-3.2.0_p20200308.ebuild]: Remove this if above is applicable.

File: sys-apps/bleachbit/bleachbit-3.2.0_p20200308.ebuild [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	$(python_gen_cond_dep '
+	dev-python/chardet[${PYTHON_MULTI_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_MULTI_USEDEP}]
+	dev-python/scandir[${PYTHON_MULTI_USEDEP}]') "
```
Comment (tietzc) [inline sys-apps/bleachbit/bleachbit-3.2.0_p20200308.ebuild]: scandir has been included in Python 3.5+. You can most likely drop it.

Comment (tietzc) [inline sys-apps/bleachbit/bleachbit-3.2.0_p20200308.ebuild]: Minor nit. Please make this consistent with BDEPEND, i.e. move quotation mark to next line.

**Comment by lg188** [comment]
I implemented most of the feedback in the ebuild and it's almost ready. Though I did get stuck during testing and haven't touched it since. Testing the different python version and making sure the dependencies are installed is a bit more confusing than I thought.

**Comment by tietzc** [comment]
Thanks for adressing my concerns. In terms of tests: https://dev.gentoo.org/~mgorny/python-guide/distutils.html and recent commits in the Gentoo repo. Maybe @juippis can help further.

**Comment by lg188** [comment]
Due to some school situations, I won't be able to add tests this in the near future.

**Pull request closed without merging**